### PR TITLE
feat: Implement PayFort Payment Processor

### PR DESCRIPTION
**Pull request by shadinaif**
feat: Implement PayFort Payment Processor

### CODE REVIEW

File: settings/payfort.py [hunk 1]
```diff
@@ -1,3 +1,28 @@
 from ecommerce.settings.test import *
 
 INSTALLED_APPS += ['ecommerce_payfort']
+
+PAYMENT_PROCESSOR_CONFIG = {
+    'edx': {
+        'payfort': {
+            # 'access_token': '1234',
```
Comment (OmarIthawi) [inline settings/payfort.py]: it's all empty, no problem?

Comment (shadinaif) [reply settings/payfort.py]: these are for the tests, I'm working on it 👍🏼

**Comment by OmarIthawi** [comment]
Thanks @shadinaif! Will review it in the next days.

**Comment by OmarIthawi** [comment]
Thanks @shadinaif for the tremendous effort here :)

Please mark as ready for review when you're done.

**Comment by shadinaif** [comment]
@OmarIthawi version [0.1.0](https://github.com/Zeit-Labs/ecommerce-payfort/releases/tag/v0.1.0) is tagged and ready for merge

**Comment by OmarIthawi** [comment]
Thanks a ton @shadinaif :), this is a huge effort that's finally being delivered.


Please add a nec staging pull request for this. We need to ensure it works with the eox-tenant.

**Review by iamjazzar — APPROVED** [review]
LGTM 👍

**Pull request merged**